aiohttp/http_parser.py

```diff
@@ -383,34 +390,33 @@ def get_content_length() -> Optional[int]:
                                 auto_decompress=self._auto_decompress,
                                 lax=self.lax,
                             )
+                        elif (
+                            not code_indicates_empty_body
```

Comment:
I'm thinking in both PRs, maybe `must_be_empty_body` feels a little clearer to me when reading the if statements.

Reply:
adjusted in 68c5bc2fbceb39a04c7b7abeaa4509bdac989eea